tests/test_messaging.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import helper
+from pythonwhat.Reporter import Reporter
+
+def message(output, patt):
+    return Reporter.to_html(patt) == output['message']
+
+def lines(output, s, e):
+    if s and e:
+        return output['column_start'] == s and output['column_end'] == e
+    else:
+        return True
+
+@pytest.mark.parametrize('stu, patt, cols, cole', [
+        ('', 'Did you call `round()`?', None, None),
+        ('round(1)', 'Check your call of `round()`. Did you specify the second argument?', 1, 8),
+        ('round(1, a)', 'Check your call of `round()`. Did you correctly specify the second argument? Rerunning it generated an error: `name \'a\' is not defined`.', 10, 10),
+        ('round(1, 2)', 'Check your call of `round()`. Did you correctly specify the second argument? Expected `3`, but got `2`.', 10, 10)
```

Comment:
I am not in favor of the last statement. 

> Expected `3`, but got `2`. 

Not only here, but whenever we use `Ex().check_object().has_equal_value()` the default message gives away the answer. Can we end the message at "Did you correctly specify the second argument?"

(this is how we do it in `testwhat`)

Reply:
Unfortunately, the architecture of `pythonwhat` is slightly more complicated than this. If OK with you, I will not tackle this in this PR yet. When I can lock out some time for this, I'd be happy to discuss this with you in more detail before I make any adjustments. OK?

Reply:
Absolutely! As long as you agree :D